snooty/semanticparser.py

```diff
@@ -160,6 +161,25 @@ def breadcrumbs(self, pages: Dict[FileId, Page]) -> Dict[str, SerializableType]:
 
         return self.parent_paths
 
+    def toctree_order(self, pages: Dict[FileId, Page]) -> Dict[str, SerializableType]:
```

Comment:
Definitely my bad for not requesting them all along, but I think a docstring would be useful here to clarify the purpose of this function.

Reply:
No worries! On it 😃